### ISSUE TITLE
Feature request: Text 'field'

### ISSUE THREAD

**Issue by GregorKopka** [stephenmcd/django-forms-builder#161]
It would be helpful if there would be a standard field type not associated with any kind of input.

Use case is to create intermediate text sections inside the form to structure it nicer:
```
I'm interested in :
[ ] This
[ ] That

Please post more about :
[ ] Something
[ ] Else
```

**Comment by stephenmcd** [comment]
That's the checkboxes field.

**Comment by GregorKopka** [comment]
I ment the solution linked in https://github.com/stephenmcd/django-forms-builder/issues/158, could this go into core?

The checkboxes field also has some downsides compared to using individual checkbox fields (export format for one).

**Comment by stephenmcd** [comment]
Then why open a duplicate issue? Don't do that.

**Comment by GregorKopka** [comment]
When i posted this i had just installed the current version and i saw no open issue about this.

I really didn't expect that a form builder would not have the ability to create a separating paragraph (or section title) inside the form to group the contents in a logical way (since even the ones i worked with >20 years ago when everything was text-mode in a terminal had that), would need a single-trick addon to be installed (which isn't mentioned in the docs) to add such a basic thing or would even have a closed feature request about that without implementing it.

So sorry for opening up a new issue about this.

Bottom line: While the add-on linked in #158 add a paragraph field this should be in core.
Since the add-on doesn't do any more than this anyway i will make you a pull request to move this into core, if you're interested.
